### PR TITLE
hello cmake

### PR DESCRIPTION
**Pull request by yeled**
cc: #16 https://github.com/astroidmail/astroid/pull/428

### CODE REVIEW

**Comment by c-alpha** [comment]
Thanks, done (#18).

**Review by c-alpha — APPROVED** [review]

File: Formula/astroid.rb [hunk 1]
```diff
@@ -29,11 +29,13 @@ def install
...
       "-DDISABLE_TERMINAL:BOOL=OFF",
       "-DENABLE_PROFILING:BOOL=OFF",
-      "-H.",
-      "-Bbuild",
-      "-GNinja",
+      "-GUnix\ Makefiles",
     ]
-    system "cmake", *args
+    args += [
+      "--release=git",
```
Comment (gauteh) [inline Formula/astroid.rb]: Does this still have effect?

File: Formula/astroid.rb [hunk 2]
```diff
@@ -29,11 +29,13 @@ def install
...
+      "-GNinja",
```
Comment (gauteh) [inline Formula/astroid.rb]: Isn't two -G.. directives added now?

File: Formula/astroid.rb (hunk 1 above)
Comment (yeled) [reply Formula/astroid.rb]: oops, I will fix this.

File: Formula/astroid.rb (hunk 2 above)
Comment (yeled) [reply Formula/astroid.rb]: yes but the hack is that the last one takes precedence

Comment (gauteh) [reply Formula/astroid.rb]: Ok

**Pull request merged**